app/Http/Requests/WebhookUpdateRequest.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace App\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+use Illuminate\Validation\Rule;
+use Auth;
+
+class WebhookUpdateRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
+    {
+        return true;
+    }
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array
+     */
+    public function rules()
+    {
+        return [
+            'name' => 'required|min:1|max:50',
+            'description' => 'required|min:10|max:1000|',
+            'bot_id' => 'required',
+            'room_name' => 'required',
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Http/Requests/WebhookUpdateRequest.php</br>**Line**: 33</br>Each line in an array declaration must end in a comma